README.md

```diff
@@ -60,6 +60,24 @@ visit your app at `http://localhost:4200/?fastboot=false`. If you want to turn o
 
 You can even disable serving fastboot with `ember serve` using an environment flag: `FASTBOOT_DISABLED=true ember serve`. If you have disabled building fastboot assets using the same flag as described [here](https://github.com/ember-fastboot/ember-cli-fastboot#double-build-times-and-no-incremental-builds), remember to also disable serving fastboot assets when using `ember serve`.
 
+### FastBoot Configuration
+
+When running locally using `ember serve` you can pass options into FastBoot instance via `config/fastboot.js` file.
```

Comment:
Could you please update this to mention this file should only be define in the application? Want to make sure this isn't confusing for addon authors.